test/pytest/test_data/dlrm_bs_2.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
Why is the dlrm test file part of PT 1.12 upgrade PR?

Reply:
same reason as the above.